### PR TITLE
Use bwakit to write out read group info in bwakit pipeline.

### PR DESCRIPTION
**Pull request by fnothaft**
In the bwakit pipeline, we are seeing odd Picard failures (see #151). However, in the bwakit pipeline, we can write the @RG lines when we write our initial BAM file. This commit makes that change, and eliminates the further DAG stages.

CC @jvivian for review
I'll open a ticket for rationalizing the differences between the BWA and BWAKIT pipelines.

Tagged this with `needs work` while I am doing the test run.

### CODE REVIEW

File: src/toil_scripts/batch_alignment/bwa_alignment.py [hunk 1]
```diff
@@ -382,10 +395,19 @@ def run_bwa(job, job_vars):
         return_input_paths(job, work_dir, ids, 'ref.fa.alt')
 
     if input_args['use_bwakit']:
+
+        # add a read group line
+        # this is an illumina only pipeline, therefore, hardcoding illumina is OK
+        uuid = input_args['uuid']
+        library = input_args['lb']
+        rg = "@RG\tID:%s\tLB:%s\tPL:ILLUMINA\tSM:%s" % (uuid,
```
Comment (jvivian) [inline src/toil_scripts/batch_alignment/bwa_alignment.py]: Do you not need a PU tag here?

Comment (fnothaft) [inline src/toil_scripts/batch_alignment/bwa_alignment.py]: Saw your comment on #151 and updated this. Thanks for pointing that out. It's a definite hack but... GD GATK, amirite?

**Comment by fnothaft** [comment]
This is good to go; `needs work` tag is removed.

**Comment by hannes-ucsc** [comment]
+1 but would like to give @jvivian a chance to weigh in

**Comment by fnothaft** [comment]
> +1 but would like to give @jvivian a chance to weigh in
SGTM!

**Comment by fnothaft** [comment]
Ping @jvivian for review/merge.

**Pull request merged**